### PR TITLE
PoC for automation of repo creation and initialization

### PR DESCRIPTION
**Pull request by vbaruh**
Adding bash scripts as a proof of concept of how the GraphDB repository automation will be done.
The code now is bash, but later will be migrated to python.

### CODE REVIEW

File: repo/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+The scripts are written in bash as PoC, but they will be re-written in Python
+in order to automate the process with Airflow.
+
+Script description:
+* `create-repo.sh` is responsible to create a new repository described with
+repo-config.ttl. The script also deletes the repository if such exists.
+The deletion is a hacky way for avoiding duplication of triples.
+
+* `init-repo.sh` is responsible to import all turtle and sparql files in
```
Comment (nikolatulechki) [inline repo/README.md]: wording: `ttl` files are imported, `sparql` queries are executed on the endpoint. The ontorefine `sparql` scripts import data trough sparql federation via the [ontorefine virtual sparql endpoint](https://platform.ontotext.com/ontorefine/rdfizing.html#data-integration-using-the-virtual-sparql-endpoint).

File: repo/create-repo.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+log() {
+    echo "$@" >&2
+}
+
+SCRIPT_DIR="$(dirname "$0")"
+source "${SCRIPT_DIR}/vars.sh"
+
+REPO_CONFIG="${SCRIPT_DIR}/repo-config.ttl"
+REPO_URL="${GRAPHDB_URL}/rest/repositories/${REPO_ID}"
```
Comment (nikolatulechki) [inline repo/create-repo.sh:16]: If the repo name is in the config ttl file the  rest is redundant and conflicts if the name from the scrit conflicts with the one in the ttl.

Comment (vbaruh) [reply repo/create-repo.sh:16]: agree, I made the repo-config.ttl file to be a template. The script now uses `sed` to fill in the REPO_ID in the template.

File: repo/README.md (hunk 1 above)
Comment (vbaruh) [reply repo/README.md]: Thanks for the tips. 
I changed the text a bit, hopefully it's better now.

**Pull request merged**